sherpa/optmethods/tests/test_opt_unit.py

```diff
@@ -144,3 +144,22 @@ def test_optmethod_setattr(cls):
     # which would mean changing newval
     #
     assert oldval != pytest.approx(newval)
+
```

Comment:
I'm trying to have a consistent approach where we have two blank lines between top-level functions. I know we don't (yet?) have a style guide setting this out in stone (and I'm not suggesting adding one for this PR). I'm not sure what black/ruff/whatever use.

Reply:
Yes, two empty Ines are recommended by PEP 8.